models/llama4/model.py

```diff
@@ -221,8 +210,8 @@ def forward(
             xq, xk = apply_rotary_emb(xq, xk, freqs_cis=freqs_cis)
 
         if self.use_qk_norm:
-            xq = self.qk_norm(xq)
-            xk = self.qk_norm(xk)
+            xq = rmsnorm(xq, self.norm_eps)
```

Comment:
Still use qk_norm as wrapper? Otherwise, remove RMSNorm class and just use rmsnorm function?

Reply:
@jianyuh RMSNorm is used by the FFN with a trained weight parameter. Here we are talking about Attention